source/languages/en/dataplatform/using-dataplatform/configuration/setup-a-cluster.md

```diff
@@ -278,6 +279,11 @@ In the above example, the Redis service will use the default port 6379.
 
 ### Default Ports
 
+<div class="note">
+<div class="title">AWS Security Group Settings</div>
+If you're using AWS, be sure to check out [Installing on AWS Marketplace][aws marketplace] for instructions on configuring security group settings to work with Riak.
+</div>
```

Comment:
TIL this week... the reference-style links don't work in notes formatted this way :( They need to be 
>Note-
>style, notes.